docs/hub/webhooks-guide-auto-retrain.md

```diff
@@ -0,0 +1,176 @@
+# Webhook guide: Setup an automatic system to re-train a model when a dataset changes
+
+This guide will help walk you through the set up of an automatic training pipeline on the HuggingFace platform
+using HF Webhooks, Spaces and AutoTrain.
+
+We will build a Webhook that listens to changes on an image classification dataset and triggers a fine-tuning
+of [microsoft/resnet-50](https://huggingface.co/microsoft/resnet-50) using [AutoTrain](https://huggingface.co/autotrain).
```

Comment:
pinging @abhishekkrthakur too! this is quite cool!

For distribution/growth, we should do more of those type of API integrations using AutoTrain in the coming months IMO, cc @Michellehbn @gary149 too